airflow/providers/airbyte/hooks/airbyte.py

```diff
@@ -257,3 +257,39 @@ def test_connection(self):
             return False, str(e)
         finally:
             self.method = "POST"
+
+    def get_airbyte_connection_info(self, connection_id: str):
+        """
+        Get the connection info for Airbyte.
+
+        :return: The connection info for Airbyte.
+        """
+        conn = self.get_connection(self.http_conn_id)
+
+        res = self.run(
+            endpoint=f"api/{self.api_version}/connections/get",
+            headers={"accept": "application/json", "authorization": f"Bearer {conn.password}"},
+            json={
+                "connectionId": connection_id,
+            },
+        )
+
+        return res
+
+    def get_airbyte_destination(self, destination_id):
```

Comment:
Same as above

Reply:
good point :) best I can do for the result type is Any as the parent class is returning that.